data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+				`(Wait for the traitor near the exit to stun them.)`
+					goto wait
+				`(Order the crew member to stand down.)`
+					goto shout
+				`(Shoot.)`
+			label shoot
+			`You hit the crew member in the shoulder and they fall. You go remove their weapon to be safe and ask for an explanation, but the only answer you get is "nemo me impune lacessit" before they lose consciousness. Looking eye in the eye, you could see terror. You put them at a nearby hospital, but when you come back later, the crew member is gone, and no one has seen or heard anything.`
+			action
+				set "unfettered: knows about bars"
+			``
+				defer
+			label hello
+			`In order to be sure to avoid any misunderstanding, you shout "Hello?" to see if someone answers. Nobody does. You still head into the cargo bay, a bit more alert than before. As you enter you see a few gold bars have fallen off the pile. That was probably the sound you heard.`
+			choice
+				`(Head in that direction.)`
+				`(Call for help.)`
+					goto help
+			`You move that way, but soon you feel a pressure on the back of your neck. "Sweet dreams, Captain." You recognize the voice of one of your crew before you are incapacitated by a quick hit to your head. When you wake up, some of the bars are missing.`
+			label lostbars
+			action
+				outfit "Gold Bars" -5
+				set "unfettered: knows about bars"
+			``
+				defer
+			label wait
+			`You patiently wait for the intruder to arrive and hit them in the back with your pistol. Given it is a member of your crew that betrayed you, you decide not to involve anyone else, as they could be complicit. You first remove the weapon they are carrying. After doing a thorough search, you notice a distinct marking on the crew member's leg, which is only visible by a special ultraviolet light. This seems bad, very bad. Then you pause for a second to wonder if you really want to know about this.`
+			choice
+				`(Ignorance is bliss.)`
+				`(I prefer not to stay in the dark. Wake them.)`
+					goto light
+			label bliss
+			`You leave the traitor here on this world after awakening them, and they seems thankful, but deeply troubled by what happened. You wonder what you should do with the gold bars that seem to cause so much trouble. They should probably be stashed safely, or maybe you should get rid of them.`
+			action
+				set "unfettered: knows about bars"
+			``
+				defer
+			label light
+			`You throw a bucket of water on the crew member. He then wakes up, and at first refuses to tell you anything, but once you reveal you're aware of their marking, he agrees to reveal something to you. He seems to recognize the hopelessness of his situation, and maybe willing to bargain. "This sign means that I holds my life to someone, and have to do one task, any task, to repay them. There is something here, on this ship, of the highest importance to some... I may not, and will not say anything else, no matter what you try to make me do it. I am dead already, and they own me."`
```

Comment:
Idk if I can give a name this mission can happen multiple times

The option to help them will come later, right now you dont know enough (as in, next PR) but I should probably mention that

They dont want your help, it means your death and theirs as well in their eyes

Reply:
I'm not sure I like the idea of missions that feel unique like this one being able to happen multiple times without any variety. If it felt less "personal" then maybe I would be fine with it, but seeing this exact same text on multiple instances feels weird to me. Is there any way this could be reworked into just the first instance, then subsequent missions are just short blurbs about something similar happening? Maybe those blurbs change depending on our choices here?

Reply:
Seems good, should probably be along the lines of the same crew members will try again, until it gets too hot for them